io-ballerina/src/io/bytes/readable_byte_channel.bal

```diff
@@ -62,19 +66,19 @@ public class ReadableByteChannel {
         return new stream<Block>(blockStream);
     }
 
-    # Encodes a given `ReadableByteChannel` using the Base64 encoding scheme.
+    # Encodes a given `ReadableByteChannel` using the Base64 encoding scheme
     # ```ballerina
-    # ReadableByteChannel|Error encodedChannel = readableByteChannel.base64Encode();
+    # io:ReadableByteChannel|Error encodedChannel = readableByteChannel.base64Encode();
     # ```
     #
     # + return - An encoded `ReadableByteChannel` or else an `io:Error`
     public function base64Encode() returns ReadableByteChannel|Error {
         return base64EncodeExtern(self);
     }
 
-    # Decodes a given `ReadableByteChannel` using the Base64 encoding scheme.
+    # Decodes a given `ReadableByteChannel` using the Base64 encoding scheme
```

Comment:
Is it correct to say this? Concerned about the use of "encoding scheme" in the decoding context. Did a quick search and it doesn't seem to be referred to like that. Shall we just simply say that this decodes a given Base64-encoded `ReadableByteChannel`?

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8